src/main/proto/contract.proto

```diff
@@ -0,0 +1,92 @@
+syntax = "proto3";
+package proto;
+
+option java_multiple_files = true;
+option java_package = "generated";
+option java_outer_classname = "ProductsProtos";
+
+service product {
+  rpc addProduct(addProductRequest) returns (addProductResponse) {};
+  rpc listProduct(listProductRequest) returns (stream listProductResponse) {};
+  rpc listById(listByIdRequest) returns (listByIdResponse) {};
+  rpc createAShoppingCart(createAShoppingCartRequest) returns (createAShoppingCartResponse) {};
+  rpc addProductsToShoppingCart(addProductsToShoppingCartRequest) returns (addProductsToShoppingCartResponse){};
+  rpc listShoppingCartProducts(listShoppingCartProductsRequest) returns (stream listShoppingCartProductsResponse) {};
+  rpc calculateTotalAmount(calculateTotalAmountRequest)  returns (calculateTotalAmountResponse){};
```

Comment:
These methods are starting with small Letters. This will cause your classes generated by gRPC to start with small letters also. The same happens for messages.

Check this file for this kinda of error

Reply:
Done.